src/main/java/lk/gov/health/phsp/bean/ClientController.java

```diff
@@ -263,83 +262,69 @@ public void clearExistsValues() {
 
     public void checkPhnExists() {
         phnExists = null;
-        if (selected == null) {
-            return;
-        }
-        if (selected.getPhn() == null) {
-            return;
-        }
-        if (selected.getPhn().trim().equals("")) {
+        if (selected == null || selected.getPhn() == null || selected.getPhn().trim().equals("")) {
             return;
         }
+
         phnExists = checkPhnExists(selected.getPhn(), selected);
     }
 
     public Boolean checkPhnExists(String phn, Client c) {
+        Map param_map = new HashMap();
```

Comment:
Throughout the application, there are thousands of places where hashmap is used to pass parameters to JPQL. Everywhere it was used as m, along with the JPQL query identified with j. As they are method variables, there is no need to rename them. One point is that because of its narrow scope, it is easily understandable. Writing long variable name for such a narrow scope frequently used variable in so many methods is time-consuming. The code consistency is lost. I think it is better to adhere to using m for the hashmap to pass parameters in method scope. We definitely have to refactor the class variable when the meaningful names are not used.

Reply:
Ok, I just wanted to have a meaningful pull name there. I will revert changes in those places and commit again.